manifests/pulp.pp

```diff
@@ -84,6 +84,7 @@
 ) {
   include katello::params
   include certs
+  include certs::qpid
 
   class { 'certs::qpid_client':
```

Comment:
You deleted this class but still include it here.

Reply:
I dropped `katello::qpid_client`. This one handles client certificates for Pulp effectively to talk to Qpid.

Reply:
I shouldn't do reviews when I'm not fully awake ... I assumed the test failure was due to this